src/verification_key.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+var defaultKey = []byte(`
```

Comment:
this should really be loaded from a file somewhere, hardcoded into binary like this shouldn't be the way to go.... file at some /etc/ location or similar with command line flag to overwrite should be used

Reply:
ok, we will remove this, a default key can then be shipped via the deb package